tests/test_database_types.py

```diff
@@ -394,7 +442,8 @@ def _drop_table_if_exists(conn, table):
             conn.query(f"DROP TABLE {table}", None)
         else:
             conn.query(f"DROP TABLE IF EXISTS {table}", None)
-            conn.query("COMMIT", None)
+            if not isinstance(conn, db.BigQuery):
+                conn.query("COMMIT", None)
```

Comment:
Ah yeah, because of that weird problem where we don't run both 😱 

Can you add an issue to fix that?